src/Endpoint.js

```diff
@@ -146,9 +146,16 @@ class Method extends Component {
           <Heading level={2}>
             {header}
           </Heading>
-          <Markdown>
-            {(method.description || '').replace(new RegExp('</BR>', 'gi'), '\n\n')}
-          </Markdown>
+          { method && method.summary &&
+            <Markdown>
+              {(method.summary || '').replace(new RegExp('</BR>', 'gi'), '\n\n')}
```

Comment:
Don't need the `||` based on the test at line 149.

Reply:
Oops, definitely a bit overly cautious 😄